sdk/src/main/scala/com/horizen/forge/AbstractForgeMessageBuilder.scala

```diff
@@ -301,6 +301,7 @@ abstract class AbstractForgeMessageBuilder[
       collectTransactionsFromMemPool(nodeView, blockSize)
     }
 
+    log.debug(s"Transactions to apply ${transactions}")
```

Comment:
let's remove it or at most keep as `trace`

Reply:
Done